src/scores/categorical/contingency_impl.py

```diff
@@ -115,6 +115,52 @@ def accuracy(self) -> xr.DataArray:
         ratio = correct_count / count_dictionary["total_count"]
         return ratio
 
+    def base_rate(self) -> xr.DataArray:
+        """
+        The observed event frequency.
+
+        Returns:
+            xr.DataArray: An xarray object containing the base rate.
+
+        .. math::
+            \\text{base rate} = \\frac{\\text{true positives} + \\text{false negatives}}{\\text{total count}}
+
+        Notes:
+            - Range: 0 to 1, where 1 indicates the event occurred every time.
+            - "True positives" is the same as "hits".
+            - "False negatives" is the same as "misses".
+
+        References:
+            Jolliffe, I.T. and Stephenson, D.B. eds., 2012. Forecast verification: a
+            practitioner's guide in atmospheric science. John Wiley & Sons.
+        """
+        cd = self.counts
+        br = (cd["tp_count"] + cd["fn_count"]) / cd["total_count"]
+        return br
+
+    def forecast_rate(self) -> xr.DataArray:
+        """
+        The forecast event frequency.
+
+        Returns:
+            xr.DataArray: An xarray object containing the forecast rate.
+
+        .. math::
+            \\text{forecast rate} = \\frac{\\text{true positives} + \\text{false positives}}{\\text{total count}}
+
+        Notes:
+            - Range: 0 to 1, where 1 indicates the event was forecast every time.
+            - "True positives" is the same as "hits".
+            - "False positives" is the same as "false alarms".
+
+        References:
+            Jolliffe, I.T. and Stephenson, D.B. eds., 2012. Forecast verification: a
+            practitioner's guide in atmospheric science. John Wiley & Sons.
```

Comment:
In the table, for "Forecast Rate", you specifically list chapter 3 of this text. Should that be the case here too? (I honestly don't know the answer to this).

If so, should the citation be:

Hogan, R. J. & Mason, I. B. (2011). Deterministic forecasts of binary events. In I. T. Jolliffe & D. B. Stephenson (Eds.), Forecast Verification (2nd ed, pp. 39-51). https://doi.org/10.1002/9781119960003.ch3